backbone.js

```diff
@@ -983,13 +983,13 @@
     // collection when they arrive. If `reset: true` is passed, the response
     // data will be passed through the `reset` method instead of `set`.
     fetch: function(options) {
-      options = options ? _.clone(options) : {};
-      if (options.parse === void 0) options.parse = true;
-      var success = options.success;
+      options = _.extend({parse: true}, options);
```

Comment:
No, because they can specify `{ parse: false }` if they want to. And, we don't want to mutate, their `options` object (that, and `_.defaults(null, {parse: true})` will return `null`).